bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/protocol/UsedContactInfoMessage.java

```diff
@@ -50,8 +50,22 @@ public UsedContactInfoMessage(String message) throws TelenotMessageException {
         int contactHexLen = contactNameHex.length();
         int b = 0;
         while (b < contactHexLen) {
-            Integer charcode = Integer.parseInt(contactNameHex.substring(b, b + 2), 16);
-            strBuilder.append(String.valueOf(Character.toChars(charcode)));
+            String hex = contactNameHex.substring(b, b + 2);
```

Comment:
What do you want to achieve here? If I understand the code correct, you have an input string which is a hex-encoded byte-array that contains the name encoded in LATIN-1, so your input is `54FC72` and it should be transformed to `Tür`. This can be achieved in one line by 

```
String strcontact = new String(HexUtils.hexToBytes(contactNameHex), StandardCharsets.ISO_8859_1);
```

Reply:
The alarm system returns the wrong hex for umlauts, so I get the wrong character.
The input for `Tür` is `54F572`, but it converts to `Tõr` 
Because `F5` is `õ`
Maybe i could replace all wrong hex to the right and then converts the hex-encoded byte-array to string

Reply:
Oh. I didn't notice that they use the weird HD44780 encoding. 

I guess that

```
String strcontact = new String(HexUtils.hexToBytes(contactNameHex), StandardCharsets.ISO_8859_1).replace("õ", "ü").replace(...);

```

is still a bit more clear.